config/crd/bases/capi-provider.agent-install.openshift.io_agentclusters.yaml

```diff
@@ -64,10 +64,23 @@ spec:
                 description: IgnitionEndpoint store the data to of the custom ignition
                   endpoint.
                 properties:
-                  caCertificate:
-                    description: CaCertificate a CA certficate to be used when contacting
-                      the URL via https.
-                    type: string
+                  caCertificateReference:
+                    description: CaCertificateReference is a reference to the secret
+                      containing CA certificate to be used when contacting the URL
+                      via HTTPS.
+                    properties:
+                      name:
+                        description: Name is the name of the secret containing the
+                          CA certificate.
+                        type: string
+                      namespace:
+                        description: Namespace is the namespace of the secret containing
+                          the CA certificate base64 encoded.
```

Comment:
This seems strange, I think the specific information about the fact that the cert should be base64 encoded is more relevant in the description of caCertificateReference (lines 69) and not on the namespace property.